recipes/vasp/v6.5.0/mc/config.yaml

```diff
@@ -0,0 +1,7 @@
+name: vasp-mc
+store: /user-environment
+description: VASP
+spack:
+  repo: https://github.com/spack/spack.git
+  commit: v0.21.3
```

Comment:
Worth using something more recent?

Reply:
Using any newer major release causes an internal spack error when concertizing vasp. Not sure why, and I don't really have the time right now to debug. The v.021.3 release isn't very old, so I think it should be fine unless there is a known performance issue with some library version of gcc / openblas / fft.